src/org/jitsi/impl/neomedia/rtcp/NACKPacket.java

```diff
@@ -47,13 +48,84 @@
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
  * |             PID               |             BLP               |
  * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * }</pre>
  *
  * @author Boris Grozev
+ * @author George Politis
  */
 public class NACKPacket
     extends RTCPFBPacket
 {
     /**
+     * Gets a boolean indicating whether or not the RTCP packet specified in the
+     * {@link ByteArrayBuffer} that is passed as an argument is a NACK packet or
+     * not.
+     *
+     * @param baf the {@link ByteArrayBuffer}
+     * @return true if the byte array buffer holds a NACK packet, otherwise
+     * false.
+     */
+    public static boolean isNACKPacket(ByteArrayBuffer baf)
+    {
+        int rc = RTCPHeaderUtils.getReportCount(baf);
+        return isRTPFBPacket(baf) && rc == FMT;
```

Comment:
oops, my edit below was supposed to be on this comment:

"EDIT: i see now that was another place, but maybe it'd be good here too"